### PR TITLE
Adds instructions to README

### PR DESCRIPTION
**Pull request by DarsshanP**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
+# Project Summary
+
+In this project I have built an API that allows you to access and interact with a database to provide information to front-end architecture. The purpose of this was to mimic real world backend services and gain experience in that field.
```
Comment (pkaodev) [inline README.md:3]: Worth mentioning that it is a RESTful API

File: README.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+
+Here is a link to the hosted version of this API:
+[NC-Games-Hosted](https://github.com/DarthShan/NC-Games-Hosted)
```
Comment (pkaodev) [inline README.md:6]: Is this meant to be linking to this repo?  If you are saying it is hosted it should be a link to where you have hosted it (on cyclic)

File: README.md [hunk 3]
```diff
@@ -1,3 +1,43 @@
...
+
+# Running the repo
+
+To clone this repo, create a fork on github copy the code link and type into your terminal, replacing 'url_here' with the copied link:
+
+```
+git clone url_here
```
Comment (pkaodev) [inline README.md:13]: put the address to the repo here instead of url_here

File: README.md (hunk 2 above)
Comment (DarsshanP) [reply README.md:6]: Yeah I've kinda messed it up I just realised haha. I created the README on the non hosted file but then when I pushed it pushed it to the hosted one and I think it's something to do with making the server yesterday

**Comment by pkaodev** [comment]
Nice!  Just a couple of small changes then merge away.

**Pull request closed without merging**